benefits/enrollment/templates/enrollment/index.html

```diff
@@ -41,6 +41,12 @@ <h1 class="pb-lg-8 pb-4">
       $.ajax({ dataType: "script", attrs: { nonce: "{{ request.csp_nonce }}"}, url: "{{ card_tokenize_url }}" })
           .done(function() {
           $.get("{{ access_token_url }}", function(data) {
+              if (data.redirect) {
+                // https://stackoverflow.com/a/42469170
+                // use 'assign' because 'replace' was giving strange Back button behavior
+                window.location.assign(data.redirect);
```

Comment:
I'm not worried about this as blocking this PR, but...

I wonder if `window.location.replace` is the right method here instead? Do we want to support the back button when the user is supposed to wait and not try again immediately?

Reply:
I created #2119 to follow up on this